charts/camunda-platform-alpha-8.8/templates/connectors/configmap.yaml

```diff
@@ -36,23 +36,30 @@ data:
 
     camunda:
       client:
-        zeebe:
-          {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
-          {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
-          rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
-          grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        {{-  $proto := (lower .Values.core.readinessProbe.scheme) -}}
+        {{- $baseURLInternal := printf "%s://%s" $proto (include "core.fullname" . | trimAll "\"") }}
+        enabled: true
+        rest-address: {{ printf "%s:%v%s" $baseURLInternal .Values.core.service.httpPort .Values.core.contextPath }}
+        grpc-address: {{ printf "%s:%v" $baseURLInternal .Values.core.service.grpcPort }}
+        worker:
           defaults:
             worker-threads: 10
```

Comment:
💭 I don't know if I'm looking at the wrong things here and I know you didn't add it, but when I check [the code](https://github.com/camunda/camunda/blob/b715945e764a9e132b158dbd89d86caf66538e14/clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/annotation/value/JobWorkerValue.java#L24-L38), `worker-threads` doesn't even appear to be an option 😅

Reply:
😅  it's indeed not there, I will look into this further while we are here, I can only imagine it was supported in the community sdk at some point 🤷